sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -202,6 +232,39 @@ private Message createDispositionMessage(UUID[] lockTokens, DispositionStatus di
         return message;
     }
 
+    private Flux<Instant> renewMessageLock(UUID[] renewLockList) {
+
+        return  isAuthorized(PEEK_OPERATION_VALUE).thenMany(createRequestResponse.flatMap(channel -> {
+            UUID[] lockTokens = Arrays.stream(renewLockList)
+                //.map(ServiceBusReceivedMessage::getLockToken)
+                .toArray(UUID[]::new);
+
+            Message requestMessage = createManagementMessage(REQUEST_RESPONSE_RENEWLOCK_OPERATION,
+                channel.getReceiveLinkName());
+
+            requestMessage.setBody(new AmqpValue(Collections.singletonMap(REQUEST_RESPONSE_LOCKTOKENS, lockTokens)));
+            return channel.sendWithAck(requestMessage);
+        }).flatMapMany(responseMessage -> {
+            int statusCode = RequestResponseUtils.getResponseStatusCode(responseMessage);
+            List<Instant> expirationsForLocks = new ArrayList<>();
+            if (statusCode ==  REQUEST_RESPONSE_OK_STATUS_CODE) {
```

Comment:
What if the status isn't OK? We should return an error.

Reply:
You need to run Alt+Shift+L to reformat the code. There is incorrect spacing on a few lines.